src/Connections/BaseConnection.ts

```diff
@@ -18,4 +18,9 @@ export abstract class BaseConnection {
     public get priority(): number {
         return -1;
     }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    public conflictsWithCommandPrefix(commandPrefix: string) {
```

Comment:
Thought: If this is effectively optional, why not make the function implementation optional and just call it with `conflictsWithCommandPrefix?.()`. This would allow the caller to decide what to do with a missing implementation, and avoids the eslint hacks.

Reply:
Good idea, will do